docs/xxshownativestacksymbols.md

```diff
@@ -0,0 +1,44 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]ShowNativeStackSymbols
+
+This option controls whether Java&reg; dumps show the function names of the native stack symbols in native call stacks.
+
+## Syntax
+
+        -XX:-ShowNativeStackSymbols
+        -XX:+ShowNativeStackSymbols=<value>
+
+| Setting                    | Value  |Effect|Default|
+|----------------------------|--------|------|:-------:|
+|`-XX:-ShowNativeStackSymbols`|       |Don't show native stack symbols in response to a signal, such as a fault signal      |   |
+|`-XX:+ShowNativeStackSymbols`| basic| Show easily acquired native stack symbols only     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>|
+|`-XX:+ShowNativeStackSymbols`| all| Show all the available native stack symbols     |   |
```

Comment:
Other tables like this have spaces around the content in each cell, for example:
```
| `-XX:+ShowHiddenFrames`  | Enable  |                                                                                      |
```

Reply:
This suggestion has not been adopted. Please do, unless you have a good reason it should not be done.
Then, please rebase and squash.

Reply:
Hi Keith, I checked the other topic tables and saw that the spaces in the cells are not consistent across. Also, I could see that in the generated content, the spaces in this table are same as the spaces in the other topic tables. Consistency in the spacing is being taken care of automatically at the time of generation irrespective of the number of spaces we put in the topic? It is with that belief that I didn't change the number of spaces in all the cells.

Reply:
I looked at more than twenty files and the vast majority have spaces separating cell contents from the column boundaries (`|`); please do add those spaces.

Reply:
As advised, I have added the spaces. Hope it is fine now. I have also squashed the commits.
Thanks!